build.psm1

```diff
@@ -19,6 +19,8 @@ try {
     catch { }
 }
 
+$dotnetRequiredVersion = "2.0.0-preview1-005952"
```

Comment:
It's the version of the `.NET Command Line Tools`, not the .NET Core version. Could you please update the name to be `dotnetCLIRequiredVersion`?